### DISCUSSION TITLE
Completed download behavior

### DISCUSSION THREAD

**Discussion by motif82** [crazy-max/docker-rtorrent-rutorrent#328] [Q&A]
HI, 

I am trying to transition from the old Bin-Hex docker container to Crazy-Max in Unraid and am having some issues with what is happening with the completed download folders.

 In the old container it would download files to an "incomplete" folder and then move to a "completed" folder for all my arr downloads. If I had a manually downloaded file and saved it in a different folder other than "incomplete" say "Movies", nothing would change with the save path. All pretty straight forward.

What I'm finding with the new container, everything downloaded is going into a "complete" folder with a subfolder based on the torrent tag no matter what the original path was. For example if I setup a torrent XYZ to go into my /movie folder, upon downloading it's now going to /complete/movie instead of staying in the /movie folder. This is causing me some headaches with files that I moved over that might be missing a .nfo or .srt file after rechecking. I don't really want them to move if another torrent is counting on the main files for seeding. 

Is this normal behavior or did I miss a setting when I got this up and running that is causing it?

**Comment by prilly-dev** [comment]
i have the same issue, i would love a way to disable this behaviour

**Comment by hitboxr** [comment]
I believe you would need to modify this part of `/data/rtorrent/.rtorrent.rc`:
https://github.com/crazy-max/docker-rtorrent-rutorrent/blob/55eb571804cc4406add3db3bb59bea59b3ae38f2/rootfs/tpls/.rtorrent.rc#L57-L60

I haven't tested it but taking the `,$d.custom1=` off the end of the first line should have the desired effect of not adding the label subdirectory. Personally I just commented out the whole thing and made my own automove setup because I find rtorrent's scripting environment to be incredibly obtuse at best and utterly indecipherable at worst.
